server/websocket_connector_test.go

```diff
@@ -19,7 +19,7 @@ var aTestMessage = &guble.Message{
 	Body: []byte("Test"),
 }
 
-func Test_WSHandler_SubscribeAndUnsubscribe(t *testing.T) {
+func TestWebSocket_SubscribeAndUnsubscribe(t *testing.T) {
```

Comment:
So far the convention was: Test_<Type>_<Description>
Changing to Test<Type>_<Description> would be fine for me, but than should be done consistently.